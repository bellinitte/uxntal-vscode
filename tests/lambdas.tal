{ }
.{ }
,{ }
;{ }
:{ }
-{ }
={ }
_{ }
!{ }
?{ }

{ x } x
{x x }x x
x{ x} x
