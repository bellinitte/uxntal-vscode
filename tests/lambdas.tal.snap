>{ }
#^ source.tal punctuation.braces.uxntal
# ^ source.tal
#  ^ source.tal punctuation.braces.uxntal
>.{ }
#^ source.tal punctuation.rune.uxntal
# ^ source.tal punctuation.braces.uxntal
#  ^ source.tal
#   ^ source.tal punctuation.braces.uxntal
>,{ }
#^ source.tal punctuation.rune.uxntal
# ^ source.tal punctuation.braces.uxntal
#  ^ source.tal
#   ^ source.tal punctuation.braces.uxntal
>;{ }
#^ source.tal punctuation.rune.uxntal
# ^ source.tal punctuation.braces.uxntal
#  ^ source.tal
#   ^ source.tal punctuation.braces.uxntal
>:{ }
#^ source.tal punctuation.rune.uxntal
# ^ source.tal punctuation.braces.uxntal
#  ^ source.tal
#   ^ source.tal punctuation.braces.uxntal
>-{ }
#^ source.tal punctuation.rune.uxntal
# ^ source.tal punctuation.braces.uxntal
#  ^ source.tal
#   ^ source.tal punctuation.braces.uxntal
>={ }
#^ source.tal punctuation.rune.uxntal
# ^ source.tal punctuation.braces.uxntal
#  ^ source.tal
#   ^ source.tal punctuation.braces.uxntal
>_{ }
#^ source.tal punctuation.rune.uxntal
# ^ source.tal punctuation.braces.uxntal
#  ^ source.tal
#   ^ source.tal punctuation.braces.uxntal
>!{ }
#^ source.tal punctuation.rune.uxntal
# ^ source.tal punctuation.braces.uxntal
#  ^ source.tal
#   ^ source.tal punctuation.braces.uxntal
>?{ }
#^ source.tal punctuation.rune.uxntal
# ^ source.tal punctuation.braces.uxntal
#  ^ source.tal
#   ^ source.tal punctuation.braces.uxntal
>
>{ x } x
#^ source.tal punctuation.braces.uxntal
# ^ source.tal
#  ^ source.tal entity.name.function.uxntal
#   ^ source.tal
#    ^ source.tal punctuation.braces.uxntal
#     ^ source.tal
#      ^ source.tal entity.name.function.uxntal
>{x x }x x
#^^ source.tal punctuation.braces.uxntal
#  ^ source.tal
#   ^ source.tal entity.name.function.uxntal
#    ^ source.tal
#     ^^ source.tal punctuation.braces.uxntal
#       ^ source.tal
#        ^ source.tal entity.name.function.uxntal
>x{ x} x
#^^ source.tal entity.name.function.uxntal
#  ^ source.tal
#   ^^ source.tal entity.name.function.uxntal
#     ^ source.tal
#      ^ source.tal entity.name.function.uxntal
>